invesalius/gui/widgets/canvas_renderer.py

```diff
@@ -733,9 +847,15 @@ def draw_arc(self, center, p0, p1, line_colour=(255, 0, 0, 128), width=2):
         gc.StrokePath(path)
         self._drawn = True
 
-    def draw_polygon(self, points, fill=True, closed=False, line_colour=(255, 255, 255, 255),
-                     fill_colour=(255, 255, 255, 255), width=2):
-
+    def draw_polygon(
```

Comment:
list of tuple of two integers [(X, Y)]
close is bool
line_colour is tuple of 4 integers
fill_colour is tuple of 4 integers
width is integer